src/webhook/mutation/pod_mutator/init_container.go

```diff
@@ -27,22 +28,42 @@ func createInstallInitContainerBase(webhookImage, clusterID string, pod *corev1.
 			{Name: config.K8sNamespaceEnv, ValueFrom: kubeobjects.NewEnvVarSourceForField("metadata.namespace")},
 			{Name: config.K8sNodeNameEnv, ValueFrom: kubeobjects.NewEnvVarSourceForField("spec.nodeName")},
 		},
-		SecurityContext: copyUserContainerSecurityContext(pod),
+		SecurityContext: copyUserContainerSecurityContext(pod, &dynakube),
 		Resources:       *dynakube.InitResources(),
 	}
 }
 
-func copyUserContainerSecurityContext(pod *corev1.Pod) *corev1.SecurityContext {
+func copyUserContainerSecurityContext(pod *corev1.Pod, dynakube *dynatracev1beta1.DynaKube) *corev1.SecurityContext {
```

Comment:
we don't want to copy the container security context anymore, but only ever use the lowest possible privileges. user/group should be copied, but rest of the other security context can be ignored!